lib/tasks/mail_tasks.rake

```diff
@@ -0,0 +1,17 @@
+namespace :mail_tasks do
+  desc "rake task for heroku schedule to send out the weekly notifications."
+  task send_weekly_notifications: :environment do
+    if DateTime.now.wday == 1   #Heroku schedule only allows for every day task not every week so this ensures its only sent of monday
```

Comment:
Wow. I'm really surprised Heroku doesn't let you use cron-based scheduling.

Ideally, I think I'd want to receive an email on Sunday night so that I see it at the start of the day and don't forget to unload the dishes on Monday morning.

`if Date.today.sunday?`